src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task2.java

```diff
@@ -0,0 +1,23 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your numbers please: ");
```

Comment:
To make it more user friendly, you could change the message like:
Enter 2 numbers which would represent the start and the end range
You can change the message, the main idea is to help the user to know the meaning of the numbers to be introduced

Reply:
Done.